roles/openshift_register_nodes/library/kubernetes_register_node.py

```diff
@@ -255,7 +255,7 @@ def exists(self):
         return False
 
     def create(self):
-        cmd = ['/usr/bin/osc'] + self.client_opts + ['create', 'node', '-f', '-']
+        cmd = ['/usr/bin/osc'] + self.client_opts + ['create', '-f', '-']
```

Comment:
Really, I should default the module to using the kubernetes binaries by default and having parameter overrides to pass in the alternate binaries to use.  I'd like to get this module genericized enough eventually to be able to contribute it upstream.  In the meantime, openshift kube sounds good.